doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -589,6 +594,18 @@ The Fast Pair GATT service modifies default values of related Kconfig options to
 The service also enables the needed functionalities using Kconfig select statement.
 For details, see the :ref:`bt_fast_pair_readme` Bluetooth service documentation in the |NCS|.
 
+.. _ug_bt_fast_pair_gatt_service_subsequent_pairing:
+
+Subsequent Pairing
+==================
+
+The Fast Pair specification supports the Subsequent Pairing feature.
+Subsequent pairing is the later pairing between the previously initial-paired device and the other phone where the same Google Account is logged-in.
```

Comment:
It doesn't have to be a phone
phone -> device

Reply:
or even `Fast Pair Seeker` (`device` term often refers to Provider)

Reply:
sure, I will improve it, but I simply took this sentence from the Google specification :)

https://developers.google.com/nearby/fast-pair/fast-pair-certification-guideline#11_definition